src/views/GameRouter/index.tsx

```diff
@@ -31,8 +32,13 @@ export default function GameRouter() {
       return (
         <GameEngine>
           <GameWrapper>
-            <GameTable />
-            <PlayerHand pass="CANNOT_PASS" solo="CANNOT_SOLO" cards={player.hand} />
+            <Main>
+              <BoardWrapper>
+                <GameTable />
+              </BoardWrapper>
+              <PlayerHand pass="CANNOT_PASS" solo="CANNOT_SOLO" cards={player.hand} />
```

Comment:
Put some comment that these will be provided by the gameEngine just for visibility